public/main.js

```diff
@@ -73,14 +82,20 @@ let loginLogout = () => {
     let staplesOutput = document.querySelector('.staples-output');
     let userEmailContainer = document.querySelector('.navigation-user-email-container');
     let userEmail = document.querySelector('.navigation-user-email')
+    let savedRecipes = document.querySelector('.recipes-container')
     if (logoutButton.textContent === 'Log Out') {
+        document.querySelector('.view-saved')
+        .classList.add('hidden');
+        savedRecipes.removeChild(savedRecipes.firstChild);
```

Comment:
I believe my comment above may apply to this section as well? Although, we may only need to clear the recipes in only one of these functions?